react/components/ShippingSimulator/components/ShippingTable.js

```diff
@@ -45,14 +59,24 @@ const ShippingTable = ({ shipping }) => {
         </tr>
       </thead>
       <tbody className={styles.shippingTableBody}>
-        {slaList.map(shippingItem => (
-          <ShippingTableRow
-            key={shippingItem.id}
-            name={shippingItem.friendlyName}
-            shippingEstimate={shippingItem.shippingEstimate}
-            price={shippingItem.price}
-          />
-        ))}
+        {pricingMode === GROUPED ?
```

Comment:
I think you can simplify this by just switching between the lists depending on the value of `pricingMode` and use just one map function. Something as:

```suggestion
        {(pricingMode === GROUPED ? slaSumValuesList : slaList).map(shippingItem
```

Reply:
@icazevedo 

Corrections made.

Could you review it again, please?